web/Services/AdminService.cs

```diff
@@ -44,11 +55,47 @@ public void CancelAdminAccessRequest(BricksAndHeartsUser user)
         userRecord.HasRequestedAdmin = false;
         _dbContext.SaveChanges();
     }
+    
+    public void ApproveAdminAccessRequest(int userId)
+    {
+        var userToAdmin = _dbContext.Users.SingleOrDefault(u => u.Id == userId);
+        if (userToAdmin == null)
+        {
+            throw new Exception();

```

Comment:
Worth having some text in there for someone debugging it (same below)
```suggestion
            throw new Exception($"No user found with id {userId}");
```